app/components/CollapsibleDisplayContent/index.tsx

```diff
@@ -0,0 +1,92 @@
+import Editor from '@webkom/lego-editor';
+import '@webkom/lego-editor/dist/style.css';
+import 'react-image-crop/dist/ReactCrop.css';
```

Comment:
Why do these two need to be imported?

Reply:
Yikes, no. Thanks for spotting :eyes:

Reply:
Yikes yes***. Turns out they add styling-properties to the lego-editor :sweat_smile: